app/services/users.js

```diff
@@ -17,3 +17,15 @@ exports.updateOrCreate = record =>
   User.upsert(record).catch(err => {
     throw errors.databaseError(err.message);
   });
+
+exports.getAll = ({ limit = 5, page = 0 }) =>
+  User.findAndCountAll({
+    attributes: ['firstName', 'lastName', 'email', 'isAdmin'],
+    limit,
+    offset: page * limit,
+    order: [['id', 'ASC']]
+  })
+    .then(data => Object.assign({ page }, data))
+    .catch(err => {
```

Comment:
log error

Reply:
It is necessary to log error here? If it throws an error then express' errors middleware should handle it. Then, error middleware will log error and send the response to user. Correct me if I misunderstood something 😋.